### PR TITLE
Search & Stimulus

### PR DESCRIPTION
**Pull request by ia3andy**

### CODE REVIEW

File: src/main/java/App.java [hunk 1]
```diff
@@ -0,0 +1,13 @@
...
+import javax.ws.rs.core.Response;
+import javax.ws.rs.core.Response.Status;
+
+@Path("/")
+public class App {
+
+    @GET
+    public Response home() {
+        return Response.status(Status.TEMPORARY_REDIRECT).header("Location", "/notes").build();
+    }
```
Comment (FroMage) [inline src/main/java/App.java:13]: Probably you can do something like:

```java
public class App extends Controller {
 @Path("/")
 public void home(){
  redirect(Notes.class).notes(null, null);
 }
}
```

File: src/main/java/rest/Notes.java [hunk 2]
```diff
@@ -1,73 +1,72 @@
...
                     Templates.notes$noteList(Note.listAllSortedByLastUpdated(), id),
                     Templates.notes$noteForm(note)
                     );
         }
-        return Templates.notes(Note.listAllSortedByLastUpdated(), id, note);
+        return Templates.notes(Note.listAllSortedByLastUpdated(), id, note, null);
     }
 
-    @Path("/note/{id}/delete")
+    @Path("{id}")
```
Comment (FroMage) [inline src/main/java/rest/Notes.java]: Pretty sure this `@Path("{id}")` is optional now, you can remove it :)

Yes: https://quarkiverse.github.io/quarkiverse-docs/quarkus-renarde/dev/index.html#_methods

**Review by FroMage — APPROVED** [review]
LGTM :)

File: src/main/java/App.java (hunk 1 above)
Comment (ia3andy) [reply src/main/java/App.java:13]: Thanks

File: src/main/java/rest/Notes.java (hunk 2 above)
Comment (ia3andy) [reply src/main/java/rest/Notes.java]: Right awesome

**Pull request merged**